Kibana release notes 8.19.15 (#264856)
elastic/kibana: docs/CHANGELOG.asciidoc

include::upgrade-notes.asciidoc[]

[[release-notes-8.19.15]]
== {kib} 8.19.15

The 8.19.15 release includes the following enhancements and fixes.

[float]
[[enhancement-v8.19.15]]
=== Enhancements
Elastic Security solution::
For the Elastic Security 8.19.15 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
Kibana platform::

> * Remembers the user grid state when navigating back from edit on the **Users** page ({kibana-pull}261152[#261152]).

Reviewer: ```suggestion
* Remembers the pagination state when navigating back from an edit on the **Users** page, instead of always returning to page 1 and resetting the search ({kibana-pull}261152[#261152]).
```

[float]
[[fixes-v8.19.15]]
=== Fixes
Data ingestion and Fleet::
* Fixes YAML file downloads being truncated at the first `#` character by properly URL-encoding the content ({kibana-pull}264083[#264083]).
* Fixes Fleet Server diagnostic bundles failing to download when `elasticsearch.compression` is enabled ({kibana-pull}262394[#262394]).

> * Fixes missing sort-state announcements for screen readers on the ingest pipelines table ({kibana-pull}262226[#262226]).

Reviewer 2: ```suggestion
* Fixes missing sort-state announcements for screen readers in the **Ingest pipelines** list ({kibana-pull}262226[#262226]).
```

> * Fixes screen readers announcing duplicate row indices on the ingest pipelines table ({kibana-pull}261369[#261369]).

Reviewer 2: ```suggestion
* Fixes screen readers announcing duplicate row indices in the **Ingest pipelines** list ({kibana-pull}261369[#261369]).
```

* Fixes the inference processor form to accept the `input_output` configuration shape ({kibana-pull}260517[#260517]).

> * Fixes screen readers announcing icon labels twice in the ingest pipelines list ({kibana-pull}259185[#259185]).

Reviewer 2: ```suggestion
* Fixes screen readers announcing icon labels twice in the **Ingest pipelines** list ({kibana-pull}259185[#259185]).
```
